sdk/storage/Azure.Storage.DataMovement/tests/StartTransferUploadDirectoryTests.cs

```diff
@@ -362,47 +362,6 @@ public async Task DirectoryUpload_EmptySubDirectories()
                 destinationPrefix: dirName,
                 waitTimeInSec: 10);
         }
-
-        [Test]
-        [LiveOnly] // https://github.com/Azure/azure-sdk-for-net/issues/33082
-        public async Task DirectoryUpload_Root()
```

Comment:
Just moved this test down below in new "Root" region and extracted part of it to a helper.